src/nvim/CMakeLists.txt

```diff
@@ -740,6 +740,7 @@ if(WIN32)
   set(EXTERNAL_BLOBS_SCRIPT
     "file(MAKE_DIRECTORY \"${PROJECT_BINARY_DIR}/windows_runtime_deps/platforms\")")
   foreach(DEP_FILE IN ITEMS
+      cat.exe
```

Comment:
In the future we could maintain this as a `cat.c` source file that we build and install.